### ISSUE TITLE
Use aria-controls on combox in addition to input

### ISSUE THREAD

**Issue by aksommerville** [ng-select/ng-select#2192]
Testing accessibility with aXe Dev Tools (4.6.2, Chrome, Windows).
When the ng-select is open we get an error that the element with `role="combobox"` should have `aria-controls` with the ID of the element it controls.
A correct `aria-controls` is already present on the `input` element, but it must also appear on the `<div role="combobox">`.

Tested with ng-select 9.0.3. (looked at 10.0.3 and this code doesn't seem to have changed)
